app/serializers/api/v2/serializable_provider.rb

```diff
@@ -85,6 +85,7 @@ class SerializableProvider < JSONAPI::Serializable::Resource
 
       has_many :sites
       has_many :users
+      has_many :contacts
```

Comment:
This looks wrong

https://github.com/DFE-Digital/teacher-training-api/blob/471e479c7cf7e84de5c2dd37263cf70a19b06ba1/app/serializers/api/v2/serializable_provider.rb#L50-L68


https://github.com/DFE-Digital/teacher-training-api/blob/a97b30d65a5e2f357e696d0a5f406e6d1b144fb2/app/models/contact.rb#L14-L21

It looks like its `has_one` x 5 times
https://apidock.com/rails/ActiveRecord/Associations/ClassMethods/has_one

Reply:
We're steering away from this approach so it's more agnostic and not have to define an attribute for every contact.

Reply:
I'm going to add another ticket to remove those so it's not part of this PR.